app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -17,7 +19,11 @@ def resource_locks
       end
 
       def plan(job_invocation, host, template_invocation, proxy_selector = ::RemoteExecutionProxySelector.new, options = {})
-        action_subject(host, :job_category => job_invocation.job_category, :description => job_invocation.description, :job_invocation_id => job_invocation.id)
+        action_subject(host,
+          :job_category => job_invocation.job_category,
+          :description => job_invocation.description,
+          :job_invocation_id => job_invocation.id,
+          :job_feature_label => job_invocation&.remote_execution_feature&.label)
```

Comment:
Shouldn't we consider all features tied to the template being run? This way if I go to the UI and instead of selecting "Run all ansible roles" for a host I go into the job invocation form, select the template for running ansible roles and the host and run it, then the event wouldn't be emitted, even though roles were in fact applied to a host.

Reply:
Right, rebased.